spython/main/parse/parsers/singularity.py

```diff
@@ -277,8 +276,7 @@ def _load_bootstrap(self, line):
            exit on fail (there is no other option to convert to Dockerfile!
         '''
         if 'docker' not in line.lower():
-            bot.error('docker not detected as Bootstrap!')
-            sys.exit(1)
+            bot.exit('docker not detected as Bootstrap!')
```

Comment:
No! You are parsing a Singularity File here. Conversion does not matter at this point

Reply:
This is a part of a Singularity recipe - the "from" statement can be "docker." Are you familiar with Singularity?

Reply:
The reason we limit to docker is because the other from types need more specific checking, and allow other fields. It's most common to use a docker bootstrap and I'll adjust this line when a user comes requesting it.

Reply:
Then at least adjust the comment above and throw "NotImplementedError" (or so) if it isn't docker. The current reasoning is valid but not clear from the code.

Reply:
Sure, if you think that is more clear for the user, happy to do it.